src/files-card-body.tsx

```diff
@@ -409,6 +418,7 @@ export const FilesCardBody = ({
                               modifier="nowrap"
                             >{_("Modified")}
                             </Th>
+                            <Th sort={sortColumn(3)} className="col-perms">{_("Permissions")}</Th>
```

Comment:
Please add `modifier="nowrap"` that fixes the wrapping of the Permissions column.